public/app/features/alerting/unified/CloneRuleEditor.tsx

```diff
@@ -42,6 +42,9 @@ export function CloneRuleEditor({ sourceRuleId }: { sourceRuleId: RuleIdentifier
       formPrefill.group = '';
     }
 
+    // Since this is a new rule, we don't want to carry over the uid of the original rule
+    formPrefill.uid = '';
```

Comment:
Can we add a test for that? The thing My concern is that the cloning became broken due to changes in other components, not in this one